diff --git a/node_modules/@angular/bazel/src/external.bzl b/node_modules/@angular/bazel/src/external.bzl
index 84d37b3..a787bbb 100755
--- a/node_modules/@angular/bazel/src/external.bzl
+++ b/node_modules/@angular/bazel/src/external.bzl
@@ -2,11 +2,11 @@
 """
 
 load(
-    "@npm_bazel_typescript//internal:build_defs.bzl",
+    "@npm//@bazel/typescript/internal:build_defs.bzl",
     _tsc_wrapped_tsconfig = "tsc_wrapped_tsconfig",
 )
 load(
-    "@npm_bazel_typescript//internal:common/compilation.bzl",
+    "@npm//@bazel/typescript/internal:common/compilation.bzl",
     _COMMON_ATTRIBUTES = "COMMON_ATTRIBUTES",
     _COMMON_OUTPUTS = "COMMON_OUTPUTS",
     _DEPS_ASPECTS = "DEPS_ASPECTS",
@@ -14,7 +14,7 @@ load(
     _ts_providers_dict_to_struct = "ts_providers_dict_to_struct",
 )
 load(
-    "@npm_bazel_typescript//internal:ts_config.bzl",
+    "@npm//@bazel/typescript/internal:ts_config.bzl",
     _TsConfigInfo = "TsConfigInfo",
 )
 load(
diff --git a/node_modules/@angular/bazel/src/ngc-wrapped/_BUILD.bazel b/node_modules/@angular/bazel/src/ngc-wrapped/_BUILD.bazel
index adcfa82..e9144d0 100755
--- a/node_modules/@angular/bazel/src/ngc-wrapped/_BUILD.bazel
+++ b/node_modules/@angular/bazel/src/ngc-wrapped/_BUILD.bazel
@@ -1,5 +1,5 @@
 load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
-load("@npm_bazel_typescript//:index.bzl", "ts_library")
+load("@npm//@bazel/typescript:index.bzl", "ts_library")
 
 ts_library(
     name = "ngc_lib",
