diff --git a/node_modules/symbol-observable/index.d.ts b/node_modules/symbol-observable/index.d.ts
index 8f77836..c74bade 100644
--- a/node_modules/symbol-observable/index.d.ts
+++ b/node_modules/symbol-observable/index.d.ts
@@ -3,6 +3,6 @@ export default observableSymbol;
 
 declare global {
   export interface SymbolConstructor {
-    readonly observable: unique symbol;
+    readonly observable: symbol;
   }
 }
